src/main/java/com/angelbroking/smartapi/SmartConnect.java

```diff
@@ -1,727 +1,781 @@
-			params.put("exchange", exchange);
-
-			String url = routes.get("api.gtt.cancel");
-			JSONObject jsonObject = smartAPIRequestHandler.postRequest(this.apiKey, url, params, accessToken);
-			Gtt gtt = new Gtt();
-			gtt.id = jsonObject.getJSONObject("data").getInt("id");
-			System.out.println(gtt);
-			return gtt;
-		} catch (Exception | SmartAPIException e) {
-			System.out.println(e.getMessage());
-			return null;
-		}
-	}
-
-	/**
-	 * Get Gtt Rule Details.
-	 * 
-	 * @param id is gtt rule id.
-	 * @return returns the details of gtt rule.
-	 */
-
-	public JSONObject gttRuleDetails(Integer id) {
-		try {
-
-			JSONObject params = new JSONObject();
-			params.put("id", id);
-
-			String url = routes.get("api.gtt.details");
-			JSONObject response = smartAPIRequestHandler.postRequest(this.apiKey, url, params, accessToken);
-			System.out.println(response);
-
-			return response.getJSONObject("data");
-		} catch (Exception | SmartAPIException e) {
-			System.out.println(e.getMessage());
-			return null;
-		}
-
-	}
-
-	/**
-	 * Get Gtt Rule Details.
-	 * 
-	 * @param status is list of gtt rule status.
-	 * @param page   is no of page
-	 * @param count  is the count of gtt rules
-	 * @return returns the detailed list of gtt rules.
-	 */
-	public JSONArray gttRuleList(List<String> status, Integer page, Integer count) {
-		try {
-			JSONObject params = new JSONObject();
-			params.put("status", status);
-			params.put("page", page);
-			params.put("count", count);
-
-			String url = routes.get("api.gtt.list");
-			JSONObject response = smartAPIRequestHandler.postRequest(this.apiKey, url, params, accessToken);
-			System.out.println(response);
-			return response.getJSONArray("data");
-		} catch (Exception | SmartAPIException e) {
-			System.out.println(e.getMessage());
-			return null;
-		}
-
-	}
-
-	/**
-	 * Get Historic Data.
-	 * 
-	 * @param params is historic data params.
-	 * @return returns the details of historic data.
-	 */
-	public String candleData(JSONObject params) {
-		try {
-			String url = routes.get("api.candle.data");
-			JSONObject response = smartAPIRequestHandler.postRequest(this.apiKey, url, params, accessToken);
-			System.out.println(response);
-			return response.getString("data");
-		} catch (Exception | SmartAPIException e) {
-			System.out.println(e.getMessage());
-			return null;
-		}
-	}
-
-	/**
-	 * Logs out user by invalidating the access token.
-	 * 
-	 * @return JSONObject which contains status
-	 * 
-	 */
-
-	public JSONObject logout() {
-		try {
-			String url = routes.get("api.user.logout");
-			JSONObject params = new JSONObject();
-			params.put("clientcode", this.userId);
-			JSONObject response = smartAPIRequestHandler.postRequest(this.apiKey, url, params, accessToken);
-			return response;
-		} catch (Exception | SmartAPIException e) {
-			System.out.println(e.getMessage());
-			return null;
-		}
-	}
+
+    private static final Logger logger = LoggerFactory.getLogger(SmartConnect.class);

```

Comment:
Please use logger from lombok @slf4j

Reply:
have used @slf4j instead of private static final Logger logger = LoggerFactory.getLogger(SmartConnect.class);